web-bundle/src/main/java/com/graphhopper/replica/GtfsLinkMapper.java

```diff
@@ -38,7 +39,12 @@ public void setGtfsLinkMappings() {
 
         // Initialize mapdb database to store link mappings and route info
         logger.info("Initializing new mapdb file to store link mappings");
-        DB db = DBMaker.newFileDB(new File("transit_data/gtfs_link_mappings.db")).make();
+        File linkMappingdirectory = new File(GTFS_LINK_MAPPING_DIR);
+        if (!linkMappingdirectory.exists()){
+            linkMappingdirectory.mkdir();
+        }
+        DB db = DBMaker.newFileDB(new File(GTFS_LINK_MAPPING_DIR + "gtfs_link_mappings.db")).make();
```

Comment:
nit: maybe a little cleaner with the [`File(File parent, String child)`](https://docs.oracle.com/javase/7/docs/api/java/io/File.html#File(java.io.File,%20java.lang.String)) constructor, so you don't have to worry about path concatenation and whether `GTFS_LINK_MAPPING_DIR` includes the trailing slash

Reply:
Good call, I'll update!